pom.xml

```diff
@@ -98,6 +98,13 @@
       <artifactId>metrics</artifactId>
       <version>3.0.0</version>
     </dependency>
+    <!-- test dependencies -->
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
+      <version>4.4.1</version>
+      <scope>test</scope>
```

Comment:
AH AH AH @andresrc @jglick I just found that the dependency was required by an orphan import cc @christ66 
I will update my PR